guides/03_API_Overview.md

```diff
@@ -357,6 +357,10 @@ To receive messages on a given topic, set up a handler using the meetingSession.
 
 To unsubscribe the receive message handler, call meetingSession.audioVideo.[realtimeUnsubscribeFromReceiveDataMessage()](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#realtimeunsubscribefromreceivedatamessage).
 
-If you send too many messages at once, your messages may be returned to you with the [throttled](https://aws.github.io/amazon-chime-sdk-js/classes/datamessage.html#throttled) flag set. The current throttling soft limit for Data Messages is Rate: 100, Burst: 200. If you continue to exceed the throttle limit (hard limit: Rate: 500, Burst: 10000), then the server may hang up the connection.
+If you send too many messages at once, your messages may be returned to you with the [throttled](https://aws.github.
+io/amazon-chime-sdk-js/classes/datamessage.html#throttled) flag set. The current throttling soft limit for Data 
+Messages is Rate: 100, Burst: 200 for a meeting session. If you continue to exceed the throttle limit, then the 
```

Comment:
Can you define `Rate` and `Burst` ? whats the time window?

Reply:
Also 'is  Rate: 100, Burst: 200' is a bit clunky.  Can we just describe the limits in plain english? i.e. 'is 100 per second or 200 for a burst of one minute' (not exactly like that, since the limit is not clear to me right now :P)

Reply:
should we just say `meeting`? to me i feel like `meeting session` sounds like it could be an individual connection.

Reply:
Thanks for your feedback. Updated.